Library/Homebrew/test/rubocops/options_cop_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rubocop"
+require "rubocop/rspec/support"
+require_relative "../../extend/string"
+require_relative "../../rubocops/options_cop"
+
+describe RuboCop::Cop::FormulaAudit::Options do
+  subject(:cop) { described_class.new }
+
+  context "When auditing options" do
+    it "32-bit" do
+      source = <<-EOS.undent
+        class Foo < Formula
+          url 'http://example.com/foo-1.0.tgz'
+          option "32-bit", "with 32-bit"
+        end
+      EOS
+
+      DEPRICATION_MSG = "macOS has been 64-bit only since 10.6"\
```

Comment:
Just in case you do this in other tests too: I'd read this constant from the class itself rather than duplicating it in the test.